<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <meta http-equiv="Content-Style-Type" content="text/css" />
        <meta name="generator" content="pandoc" />
                        <title> BIO201 SNP Final Project </title>
        <style type="text/css">code{white-space: pre;}</style>
                                                <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
        <style>
html {
    min-height: 100%;
    min-width: 100%;
    font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif; 
    font-weight: 300;
    background-color: #fafafa;
}

    #center-viewport {
        margin: 0 auto;
        padding-left: 20px;
        padding-right: 20px;
        max-width: 1200px;
        padding-bottom: 20px;
    }

    img {
        max-height: 40vh;
        width: auto;
        max-width: 100%;
    }

    a {
        font-size: 8px; color: darkgreen
    }

    h1, h2, h3 {
        margin: 0;
    }

    p {
        margin: 0;
        margin-bottom: 10px;
        margin-top: 5px;
    }

    h1 {
        margin-top: 20px;
        display: inline-block;
        /* border-bottom: 2px rgba(191, 60, 60, 0.4) solid; */
    }

    h2 {
        margin-top: 10px;
    }

    h3 {
        margin-top: 20px;
    }

    body {
        margin: 0;
    }

        </style>
    </head>
    <body>
        <div style="position: sticky; position: -webkit-sticky; top: 0; height: 30px; width: 100%; background-color: #BF3C3C; margin-bottom: 20px;z-index: 10000; color: white;   display: flex; flex-direction: column; justify-content: center;">
            <div><span style='cursor: pointer; font-family: "Courier New", Courier, monospace; font-weight: 700; margin-left: 20px' onclick="window.location.href='https://taproot.shabang.cf/'">Taproot</span><span style='cursor: default; font-family: "Courier New", Courier, monospace; font-weight: 300 !important; margin-right: 20px; float:right'> <strong>BIO201</strong> SNP Final Project </span></div>
        </div>
        <div id="center-viewport">
            <p>#ret #incomplete</p>
            <div class="center">
            <hr />
            </div>
            <h1 id="snp.-project.-write-up.">SNP. Project. Write-up.</h1>
            <p>Resources: <span>[</span><span>[</span>KBxSNPPCR<span>]</span><span>]</span> <a href="https://docs.google.com/document/d/1SRQSvppoSJYlOfJvIKg3INx79Yzv5MmXcFcNG1-nIwE/edit">Instructions</a></p>
            <h3 id="init-planning">Init Planning</h3>
            <p>Outline</p>
            <ul>
            <li><p>basics</p></li>
            <li><p>function and regulation</p></li>
            <li><p>SNP effect</p></li>
            </ul>
            <p>Writing!</p>
            <h3 id="part-one">Part One!</h3>
            <p>The <em>COMT</em> gene, or catechol-O-methyltransferase, encodes the <em>COMT</em> enzyme which is responsible for breaking down neurotransmitters the brain’s prefrontal cortex. More specifically, it acts as a catalyst for the transfer of a methyl group from S-adenosylmethionine to dopamine, epinephrine, and norepinephrine. This process, called O-methylation, leads to the degradation of the aforementioned neurotransmitters. The <em>COMT</em> enzyme also effects the metabolism of exogenous substances, but that is irrelevant for the mutation at hand <a href="https://www.ncbi.nlm.nih.gov/gene/1312">citation</a>. The <em>COMT</em> gene itself is 27.22kb long and located on chromosome 22q11.2 <a href="https://pubmed.ncbi.nlm.nih.gov/1572656/">citation</a>. It has ubiquitous expression in 27 tissues, including the placenta, the adrenal, and the lung <a href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression">citation</a>. Val158Met, also known as rs4680, is a common missense mutation swapping a guanine for an adenine. It has the frequency G=0.510915, and thus, A=0.489085. Val158Met causes the <em>COMT</em> enzyme to be roughly 25% as effective compared to the wild type. Expression levels in mRNA, despite its reduced protein abundance, are not effected by Val158Met <a href="https://www.hindawi.com/journals/dm/2020/8850859/">citation</a> &amp; <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0">citation</a>. Thus, Val158Met must be located in a protein-coding region causing the <em>COMT</em> enzyme to have lower protein integrity, explaining the discrepancy between mRNA expression and protein expression. This lower protein integrity is most likely manifested as diminished thermostability of the enzyme <a href="https://pubmed.ncbi.nlm.nih.gov/7703232/">citation</a>. The higher level effect of this reduced enzyme efficacy is greatly debated, and linked to many different phenotypes. At a broad level, lower <em>COMT</em> activity leads to higher levels of catecholamines in the prefrontal cortex. The actual effect of these increased levels are not well understood. One proposed theory is the Warrior versus Worrier hypothesis, which outlines two groups of personality traits based upon the Val158Met mutation <a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">citation</a>. The “Warrior” group, defined as the wild type group with lower levels of catecholamines like dopamine, are said to have an advantage in processing aversive stimuli. They are also said to have higher pain tolerance, be less prone to stress, less exploratory, and ect. However, many of these claims are not well defined and bordering on pseudoscience <a href="https://selfdecode.com/snp/rs4680/">citation</a>. The “Worrier” group, those with the mutation, are said to have an advantage in memory and attention tasks <a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">citation</a>. The Val158Met mutation has also been linked to schizophrenia, but this claim is debated <a href="https://pubmed.ncbi.nlm.nih.gov/32931693/">citation</a>.</p>
            <div class="center">
            <hr />
            </div>
            <h4 id="citations">Citations:</h4>
            <p><em>(order of appearance)</em></p>
            <ul>
            <li><p><a href="https://www.ncbi.nlm.nih.gov/gene/1312">NCBI COMT catechol-O-methyltransferase <span>[</span> <em>Homo sapiens</em> (human) <span>]</span></a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/1572656/">Chromosomal mapping of the human catechol-O-methyltransferase gene to 22q11.1—-q11.2</a></p></li>
            <li><p><a href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression">Gene Expression – NCBI COMT catechol-O-methyltransferase <span>[</span> <em>Homo sapiens</em> (human) <span>]</span></a></p></li>
            <li><p><a href="https://www.hindawi.com/journals/dm/2020/8850859/">Lack of Association between rs4680 Polymorphism in Catechol-O-Methyltransferase Gene and Alcohol Use Disorder: A Meta-Analysis</a></p></li>
            <li><p><a href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0">Functional Analysis of Genetic Variation in Catechol-O-Methyltransferase (COMT): Effects on mRNA, Protein, and Enzyme Activity in Postmortem Human Brain</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/7703232/">Kinetics of human soluble and membrane-bound catechol O-methyltransferase: a revised mechanism and description of the thermolabile variant of the enzyme</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">Warriors versus worriers: the role of COMT gene variants</a></p></li>
            <li><p><a href="https://selfdecode.com/snp/rs4680/">Self Decode rs4680</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/32931693/">The effect of rs1076560 (DRD2) and rs4680 (COMT) on tardive dyskinesia and cognition in schizophrenia subjects</a></p></li>
            </ul>
            <h3 id="feedback-and-revisions">Feedback and revisions</h3>
            <pre><code>This is looking good so far. It is based in solid research and you&#39;re clear about what&#39;s still unknown or uncertain. I&#39;m noticing that you don&#39;t discuss any known regulators of COMT expression (transcription factors, environmental signals, etc). Try to work that in. If there are none reported, you can mention that in the write-up. I also think you can add a bit more clarity in the portion where you&#39;re discussing the integrity/stability of the mutant protein. You&#39;ve said it leads to lower abundance. Is the decreased enzyme activity related direction to enzyme function or based on lower total abundance?</code></pre>
            <p>The <em>COMT</em> gene, or catechol-O-methyltransferase, encodes the <em>COMT</em> enzyme which is responsible for breaking down neurotransmitters the brain’s prefrontal cortex. More specifically, it acts as a catalyst for the transfer of a methyl group from S-adenosylmethionine to dopamine, epinephrine, and norepinephrine. This process, called O-methylation, leads to the degradation of the aforementioned neurotransmitters. The <em>COMT</em> enzyme also effects the metabolism of exogenous substances, but that is irrelevant for the mutation at hand <a href="https://www.ncbi.nlm.nih.gov/gene/1312">citation</a>. The <em>COMT</em> gene itself is 27.22kb long and located on chromosome 22q11.2 <a href="https://pubmed.ncbi.nlm.nih.gov/1572656/">citation</a>. It has ubiquitous expression in 27 tissues, including the placenta, the adrenal, and the lung <a href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression">citation</a>. This expression is dynamically regulated during brain development and due to environmental stimuli. Despite much research into COMT regulation, the actual processes are still mostly unknown <a href="https://pubmed.ncbi.nlm.nih.gov/21095457/">citation</a>. COMT has two promoters — P2 functions constitutively, whereas P1 has tissue dependent regulation. This tissue specific regulation is most likely done by C/EBPalpha. <a href="https://pubmed.ncbi.nlm.nih.gov/8672242/">citation</a>. Val158Met, also known as rs4680, is a common missense mutation swapping a guanine for an adenine. It has the frequency G=0.510915, and thus, A=0.489085. Val158Met causes the <em>COMT</em> enzyme to be roughly 25% as effective compared to the wild type. Expression levels in mRNA, despite its reduced protein abundance, are not effected by Val158Met <a href="https://www.hindawi.com/journals/dm/2020/8850859/">citation</a> &amp; <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0">citation</a>. Thus, Val158Met must be located in a protein-coding region causing the <em>COMT</em> enzyme to have lower protein integrity, explaining the discrepancy between mRNA expression and protein expression. This lower protein integrity is most likely manifested as diminished thermostability of the enzyme, in turn leading to its reduced effectiveness <a href="https://pubmed.ncbi.nlm.nih.gov/7703232/">citation</a>. The higher level effect of this reduced enzyme efficacy is greatly debated, and linked to many different phenotypes. At a broad level, lower <em>COMT</em> activity leads to higher levels of catecholamines in the prefrontal cortex. The actual effect of these increased levels are not well understood. One proposed theory is the Warrior versus Worrier hypothesis, which outlines two groups of personality traits based upon the Val158Met mutation <a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">citation</a>. The “Warrior” group, defined as the wild type group with lower levels of catecholamines like dopamine, are said to have an advantage in processing aversive stimuli. They are also said to have higher pain tolerance, be less prone to stress, less exploratory, and ect. However, many of these claims are not well defined and bordering on pseudoscience <a href="https://selfdecode.com/snp/rs4680/">citation</a>. The “Worrier” group, those with the mutation, are said to have an advantage in memory and attention tasks <a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">citation</a>. The Val158Met mutation has also been linked to schizophrenia, but this claim is debated <a href="https://pubmed.ncbi.nlm.nih.gov/32931693/">citation</a>.</p>
            <h4 id="citations-1">Citations:</h4>
            <p><em>(order of appearance)</em></p>
            <ul>
            <li><p><a href="https://www.ncbi.nlm.nih.gov/gene/1312">NCBI COMT catechol-O-methyltransferase <span>[</span> <em>Homo sapiens</em> (human) <span>]</span></a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/1572656/">Chromosomal mapping of the human catechol-O-methyltransferase gene to 22q11.1—-q11.2</a></p></li>
            <li><p><a href="https://www.ncbi.nlm.nih.gov/gene/1312#gene-expression">Gene Expression – NCBI COMT catechol-O-methyltransferase <span>[</span> <em>Homo sapiens</em> (human) <span>]</span></a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/21095457/">The Catechol-<em>O</em>-Methyltransferase Gene: Its Regulation and Polymorphisms</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/8672242/">Characterization of the rat catechol-O-methyltransferase gene proximal promoter: identification of a nuclear protein-DNA interaction that contributes to the tissue-specific regulation</a></p></li>
            <li><p><a href="https://www.hindawi.com/journals/dm/2020/8850859/">Lack of Association between rs4680 Polymorphism in Catechol-O-Methyltransferase Gene and Alcohol Use Disorder: A Meta-Analysis</a></p></li>
            <li><p><a href="https://www.cell.com/ajhg/fulltext/S0002-9297(07)63786-0">Functional Analysis of Genetic Variation in Catechol-O-Methyltransferase (COMT): Effects on mRNA, Protein, and Enzyme Activity in Postmortem Human Brain</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/7703232/">Kinetics of human soluble and membrane-bound catechol O-methyltransferase: a revised mechanism and description of the thermolabile variant of the enzyme</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/17008817/">Warriors versus worriers: the role of COMT gene variants</a></p></li>
            <li><p><a href="https://selfdecode.com/snp/rs4680/">Self Decode rs4680</a></p></li>
            <li><p><a href="https://pubmed.ncbi.nlm.nih.gov/32931693/">The effect of rs1076560 (DRD2) and rs4680 (COMT) on tardive dyskinesia and cognition in schizophrenia subjects</a></p></li>
            </ul>
            <h3 id="part-two">Part Two!</h3>
            <p><em>infographic time.</em></p>
            <ul>
            <li><p><strong>Create an infographic that diagrams the various connections between the gene/SNP genotypic variants and the known phenotypic associations. This graphic should visually show the biological effects of the gene/protein and studied genotypes on human traits/phenotypes. It is also important to highlight the ways in which the environment affects gene expression, protein function and/or phenotype (see the example infographic for APOE below, which makes connections to diet and traumatic brain injury). You can choose to visually organize and/or lay out your graphic in a variety of formats, but make sure that the following information is included:</strong></p></li>
            <li><p>info to include</p>
            <ul>
            <li><p><strong>gene basics</strong></p>
            <ul>
            <li><p>Gene info (gene size, location of SNP, SNP variants, SNP frequency)</p></li>
            <li><p>Protein info (protein size, protein variants if known)</p></li>
            <li><p>reg</p>
            <ul>
            <li><p>estrogen?</p></li>
            <li><p>brain develpoment</p></li>
            <li><p>eviromental</p></li>
            <li><p>P2:</p>
            <ul>
            <li><p>constitutively</p></li>
            </ul></li>
            <li><p>P1:</p>
            <ul>
            <li><p>tissue dependent, C/EBPalpha, ect.</p></li>
            </ul></li>
            </ul></li>
            </ul></li>
            <li><p><strong>SNP variants to human phenotype relationships</strong></p>
            <ul>
            <li><p>reduced COMT enzyme activity</p></li>
            <li><p>warrior worrier stuff</p></li>
            </ul></li>
            <li><p><strong>Gene-environment Interactions</strong></p>
            <ul>
            <li><p>above -&gt; situations that better suite?</p></li>
            <li><p>gene / enviroment stuff? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3447184/</p></li>
            </ul></li>
            </ul></li>
            </ul>
            <h3 id="part-three.">Part <em>Three.</em></h3>
            <h4 id="outlinin">outlinin:</h4>
            <p>prompt: In this section, you should try to provide some evolutionary insight on the SNP alleles that we see for your gene in the human population. You may focus on one particular allele if you see that it is better-researched. Using actual research into the evolution of the allele(s) and/or research about gene and gene variant functions, explain why/how the SNP allele(s) you studied were maintained in the human population (in over 1% of people studied). At some point the allele first appeared as a mutation but it subsequently spread into a relatively large proportion of people; why might that have been? - Although this section should be based on gene/SNP research and your understanding of evolution, your explanations may be somewhat speculative due to the difficulty in obtaining evidence for certain evolutionary predictions/hypotheses (because environmental pressures, migrations, and random events that influenced evolution likely happened long ago). That is okay; just show us that you’re using research-backed reasoning about your allele(s) and that you have an understanding of evolutionary mechanisms. - In terms of evolutionary mechanisms, you should be thinking about possible selective pressures that may have maintained certain alleles in the human population (and disfavored others). Note that evolution typically operates over long timescales, so the selective pressures that are most likely to have played a role were operating before civilizations, agriculture, etc (with some exceptions that may have evolved more “recently”, like lactase persistence, high-altitude adaptation, and disease resistances). - It’s also important to remember the possible contributions of mechanisms like gene flow and genetic drift in getting to the allele frequencies that we see today. It would be harder to detect whether/when these happened without complex analyses of sequences, but you can still acknowledge their possible impact and explain how these mechanisms operate.</p>
            <p>allele: g -&gt; a</p>
            <p>selection factors:</p>
            <p>population makes it’s own selection factors need for warrior / worrier</p>
            <p>organism, gene, population</p>
            <p>greedy epsilon, worrier as epsilon</p>
            <p>evolution on multiple levels like genetic, epigenetic, symbolic, cultural, ect</p>
            <p>fitness landscapes of organisms but also of tech</p>
            <p>worriers are better for jumping out of local minima warriors are better for carrying out the best strategy</p>
            <p>perhaps a collection of personality-ish traits that are all being balanced</p>
            <p>research:</p>
            <blockquote>
            <p>A comparison of human and mouse COMT confirms that the amino acid at the Val/Met locus is important for COMT activity and suggests that COMT activity has decreased during the course of evolution. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/</p>
            </blockquote>
            <p>derived allele uniqe to humans</p>
            <p>https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en</p>
            <h4 id="we-write.">We write.</h4>
            <p>Not much is known about the evolution of the COMT*L allele. In the sea of speculation, only two facts emerged: COMT activity has decreased during the course of recent evolution, and COMT*L is a derived allele unique to humans <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182110/">citation</a> &amp; <a href="https://www.jstage.jst.go.jp/article/ase/121/3/121_130731/_html/-char/en">citation</a>. For the following speculation, the phenotypes said to arise from different variations of the COMT gene will be assumed true. Speculating about evolution on the organism level is relatively straightforward: this organism evolved a patch of photosensitive skin so it could tell which way was up. This level of analysis breaks down when trying to explain altruism, and thus comes speculation on the gene level: these organisms evolved so they would “lay down <span>[</span>their<span>]</span> life for two brothers or eight cousins” - <em>J.B.S. Haldane</em>. The evolution of COMT*L is not explained solely by either of these levels — instead, its evolution operates on the population level.</p>
            <p>It can be assumed that given a constant environment without changing selection factors, their is an optimal mix of so called Warriors and Worriers. On top of the hypothetically constant selection pressures from the environment, the population itself creates its own selection pressures based upon the ratio of Warriors to Worriers. If there are too many Warriors, then a selection pressure favoring Worriers would arise, and vise versa, leading the population towards the optimal ratio of Warriors and Worriers. With heavy speculation, one could suggest that there exists a whole set of personality traits that are constantly being balanced across the population.</p>
            <p>Of course, environmental selection pressures are not constant. Hence, the question becomes, what type of environment shifts the optimal ratio of Warriors to Worriers? To answer this question, one must think of evolution in terms of a fitness landscape. An organism can be stuck in a local minima and require some large mutation to jump out of it. Disregarding genetic drift, the only other mutations that would persist would be ones leading the species to the bottom of the local minima.</p>
        </div>

        <script>
            $(document).ready(function() {
                // Generate clickable links
                let content = $("#center-viewport").html().replace(/<span>\[<\/span><span>\[<\/span>\w*?<span>]<\/span><span>]/gi, function(x) {
                    <!--let docPointer = x.match(/(\w)*/);-->
                        <!--let docPointer = x.match(/KB\w*/);-->
                    docPointer = [ x.replace(/\<\/?[a-z]+\>/g, '').slice(2, -2) ]

                    if (docPointer) {
                        let url = `https://taproot.shabang.cf/relay?request=${docPointer[0]}`;
                        return `<a href="${url}"><span>[[</span><span>${docPointer[0]}</span><span>]]</span></a>`;
                    } else {
                        console.log(`Cannot parse , returning...`);
                        return x;
                    }
                })
                $("#center-viewport").html(content);
                $("img").each(function() {  
                    let src = this.src.replace(/.*?\/Users\/houliu\/Documents\/School%20Work\/2020-2021\/KnowledgeBase\//gi, "https://taproot.shabang.cf/");
                    console.log(src);
                    $(this).attr("src", src);
                });
            });
        </script>
    </body>
</html>
